index.html

```diff
@@ -150,10 +150,8 @@
 <div class="row">
   <div class="col-md-6 col-sm-6 ">
     <h3>Download D3FC</h3>
-    <p>Download the latest JavaScript distribution <a href="https://github.com/ScottLogic/d3-financial-components/tree/master/dist">from GitHub</a>, together with the associated CSS files.</p>
-    <p>You'll also need to obtain the latest version of <a href="http://d3js.org/">D3</a>, and the Facebook
-    <a href="https://github.com/facebook/css-layout">css-layout</a> utility - we use this for styling
-    charts with CSS flexbox.</p>
+    <p>Download the latest distribution (includes CSS and JavaScript) <a href="https://www.npmjs.com/package/d3-financial-components">via npm</a>. This package will pull in the required dependencies, just add <a href="http://d3js.org/">D3</a> and you are good to go.</p>

```

Comment:
I'm not sure these instructions are clear enough, it's actually quite an involved process.